constants.go

```diff
@@ -26,6 +26,7 @@ const (
     <type arch='x86_64'>hvm</type>
     <boot dev='hd'/>
     <bootmenu enable='no'/>
+    <bios useserial='no'/>
```

Comment:
@mykaul we do need serial port to debug some issues if the vm is start failing to boot so we shouldn't remove it.

Reply:
(I usually use the VNC device for the same purpose /o\ )